SampleUserControlLibrary/SampleUserControl.xaml.cs

```diff
@@ -104,6 +104,11 @@ public string SubscriptionEndpoint
             set;
         }
 
+        public void set_subscriptionPage_defaultEndpoint(string msg)
```

Comment:
1,2,4 are accepted. 
"Default" is to differentiate the setter of endpoint and the default value of endpoint